src/source/Ice/IceAgentStateMachine.c

```diff
@@ -29,28 +29,6 @@ StateMachineState ICE_AGENT_STATE_MACHINE_STATES[] = {
 
 UINT32 ICE_AGENT_STATE_MACHINE_STATE_COUNT = ARRAY_SIZE(ICE_AGENT_STATE_MACHINE_STATES);
 
-STATUS checkIceAgentStateMachine(PIceAgent pIceAgent)
```

Comment:
Could you share in the PR description or comment, why these were removed?

Reply:
Done